src/converters/index.ts

```diff
@@ -183,7 +183,7 @@ export const unions = {
 };
 
 export const inputObjects = {
-  id: "inputObjects",
+  id: "inputobjects",
```

Comment:
we could also just lowercase the converter.id when we call `getRelativeTypeUrl` -> 

```ts
return `${converter.id.}#${sluggify(baseType.name)}`
```

but this is straightforward as well?